app/views/genres/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<% @genre.movies.each do |movie| %>
```

Comment:
for a better user experience, it'd be nice to see the genre's name on this page as well, and perhaps some text saying what this next block of text actually is :)

Reply:
similar to what you added on the movie show page